run-nim

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+. ./config
+
+if [ ! -f "spec/nim/test.nim" ]; then
+  RED='\033[1;31m'
```

Comment:
The rest of these shell scripts actually use tabs for indentations, can I ask you to replace spaces here with tabs for consistency?

Reply:
Sure, my bad for not noticing.